exercise_060_cluster_base/src/main/resources/application.conf

```diff
@@ -0,0 +1,87 @@
+akka {
+
+  discovery.method = config
+
+  management.http {
+    // This will allow one to alter the state of a cluster using Akka HTTP Management
+    // Setting this option to false may create a security exposure in production
+    // environments
+    route-providers-read-only = false
+
+    //host = 127.0.0.1
+    port = 8558
+  }
+
+  loggers          = [akka.event.slf4j.Slf4jLogger]
```

Comment:
This is no longer necessary when using akka-actor-typed (https://github.com/akka/akka/blob/master/akka-actor-typed/src/main/resources/reference.conf#L54)

```suggestion
```

Reply:
Correct - That's something I had read in the release notes (That's a great improvement BTW) and realised I forgot after I submitted the PR. Will fix that.